### DISCUSSION TITLE
API.create_friendship() takes 1 positional argument but 2 were given

### DISCUSSION THREAD

**Discussion by kunal20071** [tweepy/tweepy#1740] [Questions] [answered]
![Screenshot (88)_LI](https://user-images.githubusercontent.com/90096297/147094117-11367bf1-2088-4718-81e0-b027a9de5c93.jpg)

**Comment by Harmon758** [comment, marked as answer]
[`API.create_friendship`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API.create_friendship) doesn't accept any positional arguments.
You can pass "YouTube" as the `screen_name` keyword-only argument instead.
For more information, see the [FAQ section](https://tweepy.readthedocs.io/en/v4.4.0/faq.html#why-am-i-getting-a-typeerror-about-an-api-method-taking-1-positional-argument-but-given-2) about this in the documentation.
